### ISSUE TITLE
Quickstart for both Excel and Word show content in taskpane, then content disapears

### ISSUE THREAD

**Issue by ivan7707** [OfficeDev/generator-office#350]
Hello,  I have followed the instructions here:

https://docs.microsoft.com/en-us/office/dev/add-ins/quickstarts/excel-quickstart-react

and here

https://docs.microsoft.com/en-us/office/dev/add-ins/quickstarts/word-quickstart?tabs=visual-studio

The installation works fine. When I run the commands suggested (npm start, and npm run sideload) rts, the appropriate program started, and Show Taskpane shows up as to be expected.  

**The issue** is that in the Taskpane, the content of the App.tsx only shows up briefly then disappears.  When I click reload, again it shows up very briefly, then disappears.  This happens for both the Excel and Word Add-ins.  When I click on Attach Debugger, nothing happens.

I'm not sure how to debug this issue. 

Windows 7 Pro
Excel 2016 (up-to-date)
Current Node
Visual Studio Community 2017.

When I go to https://localhost:3000/index.html this is what I see:

![image](https://user-images.githubusercontent.com/7657845/44614815-8aaba480-a7f9-11e8-836b-3466e2efcef2.png)

I was able to get the document assembly working: https://github.com/OfficeDev/Word-Add-in-DocumentAssembly 

(Side note: I really love the idea of developing with React/JS in MS Office, how cool).

**Comment by ivan7707** [comment]
I tried using another computer to see if it was my setup that was the issue.  This is a Windows 10 machine running Office 365 proplus.  Unfortunately, when I run sideload, I receive this error:

```
> office-toolbox sideload -m my-office-add-in-manifest.xml -a Excel

(node:16268) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'stack' of undefined
    at Function.EnvelopeFactory.createExceptionData (C:\Users\mgr17907\Desktop\myaddin\My Office Add-in\node_modules\applicationinsights\out\Library\EnvelopeFactory.js:132:40)
    at Function.EnvelopeFactory.createEnvelope (C:\Users\mgr17907\Desktop\myaddin\My Office Add-in\node_modules\applicationinsights\out\Library\EnvelopeFactory.js:32:40)
    at NodeClient.TelemetryClient.track (C:\Users\mgr17907\Desktop\myaddin\My Office Add-in\node_modules\applicationinsights\out\Library\TelemetryClient.js:102:44)
    at NodeClient.TelemetryClient.trackException (C:\Users\mgr17907\Desktop\myaddin\My Office Add-in\node_modules\applicationinsights\out\Library\TelemetryClient.js:55:14)
    at logRejection (C:\Users\mgr17907\Desktop\myaddin\My Office Add-in\node_modules\office-toolbox\app\office-toolbox.js:27:32)
    at C:\Users\mgr17907\Desktop\myaddin\My Office Add-in\node_modules\office-toolbox\app\office-toolbox.js:241:13
    at Generator.throw (<anonymous>)
    at rejected (C:\Users\mgr17907\Desktop\myaddin\My Office Add-in\node_modules\office-toolbox\app\office-toolbox.js:6:65)
    at <anonymous>
(node:16268) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:16268) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

I am at a loss as what to do in either case.

**Comment by ivan7707** [comment]
Everything is working on the Windows 2010 machine now.  

As an update to the Windows 2010 issue (the second one I noted), I checked everything in the setup.  I checked the version of Word under the Office 365 ProPlus plan.  It was not up-to-date even though it should keep itself current.  I logged into Office 365, and clicked on Install for the Office programs.  This updated the versions.  The add-ins are fully working now.
